### PR TITLE
GLIdeN64.cusom.ini New Validity Check Method

### PR DESCRIPTION
**Pull request by Predator82Germany**

### CODE REVIEW

File: GLideN64.custom.ini [hunk 1]
```diff
@@ -246,6 +243,18 @@ video\multisampling=0
 Good_Name=Pokemon Puzzle League (E)(F)(G)(U)
 frameBufferEmulation\detectCFB=1
 
+[POKEMON%20STADIUM%202]
+Good_Name=Pokemon Stadium 2 (E)(F)(G)(I)(J)(S)(U)
+frameBufferEmulation\copyToRDRAM=0
+frameBufferEmulation\copyDepthToRDRAM=0
+frameBufferEmulation\validityCheckMethod=1
+
+POKEMON%20STADIUM%20G&S]
```
Comment (purplemarshmallow) [inline GLideN64.custom.ini]: You forgot [ !

File: GLideN64.custom.ini [hunk 2]
```diff
@@ -246,6 +243,18 @@ video\multisampling=0
 Good_Name=Pokemon Puzzle League (E)(F)(G)(U)
 frameBufferEmulation\detectCFB=1
 
+[POKEMON%20STADIUM%202]
+Good_Name=Pokemon Stadium 2 (E)(F)(G)(I)(J)(S)(U)
```
Comment (purplemarshmallow) [inline GLideN64.custom.ini]: There's no (J) version
japanese version is named Pokemon Stadium Kin Gin (J)

Comment (Predator82Germany) [inline GLideN64.custom.ini]: Of course

File: GLideN64.custom.ini (hunk 1 above)
Comment (Predator82Germany) [inline GLideN64.custom.ini]: Argh...

Comment (purplemarshmallow) [inline GLideN64.custom.ini]: close pull request and creatre a new one?

Comment (Predator82Germany) [inline GLideN64.custom.ini]: Then I wait til gonetz merge it

Comment (purplemarshmallow) [inline GLideN64.custom.ini]: I created a new one so you can close it

Comment (purplemarshmallow) [inline GLideN64.custom.ini]: But if you make a new commit and sync it should update automatically

**Pull request closed without merging**